xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/doc/XWikiLink.java

```diff
@@ -26,7 +26,10 @@
 
 /**
  * Represent the relation between a document using a link to an entity, and the used entity.
+ * 
+ * @deprecated
```

Comment:
missing explanation?

Reply:
There is not exactly any alternative to this class in the new API. The main point of this class was to hold the Hibernate entity.

Reply:
Sounds like you could do point to the new module to look at.

Reply:
But you won't really find anything that you can use for whatever you used to be doing with XWikiLink. The closest alternative to XWikiLink is basically EntityReference.

Reply:
Well you could end up here just because you followed the API that were returning those XWikiLink. That's why I suggest to point out to the new API module. Sounds better than nothing to me.

Reply:
> you followed the API that were returning those XWikiLink

Those APIs and deprecated too and document alternative usually.